code/headers/hardware_usart.hpp

```diff
@@ -6,8 +6,7 @@
 
 namespace r2d2::usart {
     enum class usart_ports_c {
-        uart0,
-        uart1,
+        uart1 = 0,
```

Comment:
i don't now the technical details but seems strage to me to only =0 1 of them??

Reply:
I just want to make sure it starts with 0. It is probably not needed.